### DISCUSSION TITLE
Apache Server Warning

### DISCUSSION THREAD

**Discussion by unsalkorkmaz** [quasarframework/quasar#14174] [Ideas / Proposals]
Hi,
I wasted hours of my time because of this problem:
[Apache2 and /icons path behaving differently](https://serverfault.com/questions/153177/apache2-and-icons-path-behaving-differently)

`Alias /icons/ "/usr/share/apache2/icons/"` was the problem that comes with default apache. I suggest to add this warning somewhere to prevent others to waste their time

**Comment by metalsadman** [comment]
`server` i don't think this concerns Quasar.

**Comment by unsalkorkmaz** [comment]
This prevents PWA working, fyi

**Reply by metalsadman** [reply]
P.R. for it if you really want to include it in our docs.

**Reply by unsalkorkmaz** [reply]
I dont care, I wasted my time and found the solution. 
Why sharing here? Would be nice if it helps someone. 

But would I P.R. for it? nope. 
I wrote here, decision is up to management.


As of September 2022, Apache holds 31.2% of the market according to W3Techs and 22.6% according to Netcraft.
